core/src/main/kotlin/ch/ergon/dope/DopeQuery.kt

```diff
@@ -1,3 +1,11 @@
 package ch.ergon.dope
 
+import ch.ergon.dope.resolvable.expression.unaliased.type.ParameterManager
+import ch.ergon.dope.resolvable.expression.unaliased.type.collection.IteratorManager
+
 data class DopeQuery(val queryString: String, val parameters: Map<String, Any>)
+
+class DopeQueryManager {
```

Comment:
maybe move it into an own class - It has not directly something to do with DopeQuery

Reply:
And maybe move the ParameterManager and the IteratorManager in that file as well (or at least in the same package)

Reply:
I moved all classes into one file named `Manager`.